### ISSUE TITLE
No audio when playing some .avi videos

### ISSUE THREAD

**Issue by litianjun05090** [dburckh/Media3Avi#8]
info 

libavutil      57. 13.100 / 57. 13.100
  libavcodec     59. 15.102 / 59. 15.102
  libavformat    59. 10.100 / 59. 10.100
  libavdevice    59.  1.100 / 59.  1.100
  libavfilter     8. 21.100 /  8. 21.100
  libswscale      6.  1.102 /  6.  1.102
  libswresample   4.  0.100 /  4.  0.100
Guessed Channel Layout for Input Stream #0.1 : mono
Input #0, avi, from '/storage/emulated/0/BaiduNetdisk/我的资源/深入浅出计算机组成原理《更新中》等多个文件/应用软件开发技术(VB.NET)/30.avi':
  Duration: 00:38:27.00, start: 0.000000, bitrate: 93 kb/s
  Stream #0:0: Video: mpeg4 (XVID / 0x44495658), yuv420p, 800x600 [SAR 1:1 DAR 4:3], 69 kb/s, 10 fps, 10 tbr, 10 tbn
  Stream #0:1: Audio: wmav2 (a[1][0][0] / 0x0161), 22050 Hz, mono, fltp, 19 kb/s
At least one output file must be specified

Is it because wmav2 not supported ?

**Comment by litianjun05090** [comment]
I can provide the video if necessary.

**Comment by dburckh** [comment]
Yes sir, you are correct!  I currently have no support for WMA.  Unfortunately, not all devices have a WMA codec.  My Samsung phone does, but my Pixel phone does not.  See Codec Support Notes in the readme.MD for an overview of the issue and more examples.

That said, I *should* be able to add it, but I would definitely need a sample.

**Comment by litianjun05090** [comment]
You got it : https://drive.google.com/file/d/10TDp-qmI8xcqhXvYbbt48Y59YFIeximc/view?usp=drive_link

**Comment by dburckh** [comment]
So, not having much luck with this format.  I have found only 1 device that natively supports wma (Note S20).  The codec dies on the stream.  I've tried it with and without CSD.  I'm not sure if I'm not demuxing it properly or if this file has a funky stream.  Next thing I can try is building the ffmpeg audio decoder.

I don't know much about the format and the stuff I've found is fairly vague.  It keeps pointing me back to ASF.

Anyway, I'm back-burning this for a while.
